tests/lean/interactive/field_info.lean.expected.out

```diff
@@ -1,6 +1,7 @@
 {"msgs":[{"caption":"","file_name":"f","pos_col":40,"pos_line":13,"severity":"error","text":"invalid expression, `)` expected"}],"response":"all_messages"}
+{"msgs":[{"caption":"","file_name":"f","pos_col":40,"pos_line":13,"severity":"error","text":"invalid expression, `)` expected"},{"caption":"","file_name":"f","pos_col":0,"pos_line":12,"severity":"error","text":"don't know how to synthesize placeholder\ncontext:\n⊢ Type ?"}],"response":"all_messages"}
 {"message":"file invalidated","response":"ok","seq_num":0}
 {"record":{"full-id":"nat.to_string","source":,"type":"ℕ → string"},"response":"ok","seq_num":2}
 {"record":{"full-id":"list.all","source":,"type":"Π {α : Type}, list α → (α → bool) → bool"},"response":"ok","seq_num":4}
 {"record":{"full-id":"list.all","source":,"type":"Π {α : Type}, list α → (α → bool) → bool"},"response":"ok","seq_num":9}
-{"record":{"full-id":"list.all","source":,"type":"Π {α : Type u}, list α → (α → bool) → bool"},"response":"ok","seq_num":13}
+{"record":{"full-id":"list.all","source":,"type":"Π {α : Type _elab_u.16.225}, list α → (α → bool) → bool"},"response":"ok","seq_num":13}
```

Comment:
They're already leaking in quite a few places before this PR.  For example in proofs:
```lean
lemma foo (α) (xs : list α) : xs ++ [] = xs :=
begin trace_state, simp end
       -- ^ leak visible here
```
And also here when you hover over α:
```lean
def bar (α) (xs : list α) : list α := xs
```

As for why, I believe we use the auxiliary universes for implicit levels during elaboration.  Anything you print (or save in the info_manager) during elaboration will mention the auxiliary names.  We don't even substitute the universe metavariables at the moment.  I'll take a closer look at what we can do tomorrow.